virttest/qemu_monitor.py

```diff
@@ -212,7 +212,7 @@ def __init__(self, vm, name, filename, suppress_exceptions=False):
         self._passfd = None
         self._supported_cmds = []
         self.debug_log = False
-        self.log_file = "%s-%s.log" % (name, vm.name)
+        self.log_file = "%s-%s-%s.log" % (name, vm.name, vm.get_pid())
```

Comment:
When it failed to get a proper pid due to some reason, the filename would be set to something like `monitor1-vm1-None.log` which looks a little bit confusing. So I'd like to request the pattern to be `"%s-%s-pid-%s.log" % (name, vm-name, vm-pid)`, and the value of `vm-pid` should be set to `unknown` when the former happened.

Reply:
@luckyh  updated, please help review again, thanks a lot!